frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetOverview/ClusterSetOverview.tsx

```diff
@@ -18,28 +18,65 @@ import { MultiClusterNetworkStatus } from '../../components/MultiClusterNetworkS
 import { ClusterSetContext } from '../ClusterSetDetails'
 import { submarinerHealthCheck, SubmarinerStatus } from '../ClusterSetSubmariner/ClusterSetSubmariner'
 import { PluginContext } from '../../../../../../lib/PluginContext'
+import { ManagedClusterSetBindingModal } from '../../components/ManagedClusterSetBindingModal'
 
 export function ClusterSetOverviewPageContent() {
     const { t } = useTranslation()
     const { isSubmarinerAvailable } = useContext(PluginContext)
     const { push } = useHistory()
-    const { clusterSet, clusters, clusterPools, submarinerAddons, clusterSetBindings } = useContext(ClusterSetContext)
+    const { clusterSet, clusters, clusterPools, submarinerAddons, clusterSetBindings, clusterRoleBindings } =
+        useContext(ClusterSetContext)
 
     const unhealthySubmariners = submarinerAddons!.filter(
         (mca) => submarinerHealthCheck(mca) === SubmarinerStatus.degraded
     )
 
+    const navigateToClusterSet = () => {
+        if (clusterSet?.metadata?.name) {
+            push(NavigationPath.clusterSetClusters.replace(':id', clusterSet!.metadata.name!))
+        }
+    }
+
+    const [showManagedClusterSetBindingModal, setShowManagedClusterSetBindingModal] = useState(false)
+    let users = 0
+    let groups = 0
+    clusterRoleBindings?.forEach((binding) => {
+        binding.subjects.forEach((subject) => {
+            if (subject.kind === 'Group') {
+                groups += 1
+            }
+            if (subject.kind === 'User') {
+                users += 1
+            }
+        })
+    })
+    let userManagementCount = ''
+    if (users === 0 && groups === 0) {
+        userManagementCount = t('table.none')
+    } else {
+        userManagementCount =
+            t(users === 1 ? 'table.user' : 'table.user_plural', { count: users }) +
+            ', ' +
+            t(groups === 1 ? 'table.group' : 'table.group_plural', { count: groups })
```

Comment:
```suggestion
            t('table.user', { count: users }) +
            ', ' +
            t('table.group', { count: groups })
```

Translation support will automatically use the right key.

Reply:
🤦🏽‍♂️ glossed over that...